tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ng.html

```diff
@@ -168,7 +168,11 @@
     </table>
   </ng-template>
 </div>
-
+<ng-container *ngIf="selectedTime">
```

Comment:
This is fine now but later we want to decouple the table and linked time. Might worth adding TODO or comment here